### ISSUE TITLE
You didnt describe your process

### ISSUE THREAD

**Issue by Razpudding** [niyorn/fe3-assessment-2#5]
Also the data desc is very minimal. I cant tell from the readme how you cleaned code or added interaction. You also dont describe features or a license.

**Comment by niyorn** [comment]
Yeah, my bad. Didnt have time to write it. It was choosing between writing the readme or finishing the code.
